flaskr/blueprints/common.py

```diff
@@ -17,44 +17,12 @@ def dashboard(name='dashboard'):
     print("CURR", flask_login.current_user)
     return render_template('dashboard.html', name=name)
 
-@common.route('/classpage', methods=["GET", "POST"])
-@login_required
-def classpage(name='classpage'):
-    return render_template('classpage.html', name=name)
-
-@common.route('/assignmentpage', methods=["GET", "POST"])
-@login_required
-def assignmentpage(name='assignmentpage'):
-    return render_template('assignmentpage.html', name=name)
 
-@common.route('/assignments',  methods=['GET'])
+# Routes to the all_assignments page
+@common.route('/all_assignments',  methods=['GET'])
```

Comment:
same here. we should have `assignments` instead of `all_assignments` since it's a publicly exposed url.